docs/source/RECOMMENDATIONS.md

```diff
@@ -0,0 +1,34 @@
+[How (Not) to Generate a Highly Predictive Biomarker Panel Using Machine Learning](https://pubs.acs.org/doi/10.1021/acs.jproteome.2c00117). This effect will not occur in the default settings as the feature selection will be applied on each cross-validation split.
+
+### Sample Size
+The sample size for ML analysis is a crucial parameter to get meaningful performance metrics. In the OmicLearn manuscript, we motivate the use of ML as proteomics reaches hundreds to thousands of samples, which should be seen as a minimum number.
+In principle, we already risk overfitting in an ML context when we have more features than samples, which could easily be the case for a study with hundreds of patients and thousands of detected proteins.
+Another aspect is that the reported metrics can become unreliable. This can be exemplified with some example calculations.
+Consider the case of 100 samples: When using the default CV split of 5, we train on 80 samples and validate on 20. Only one misclassification would lead to 18 of 20 samples being correct, and achieve an accuracy of 90% - so a magnitude of 10%.
+
+### Interpretation of results
+Users should be advised that the performance values are reported on the cross-validation of training and validation data and not on the holdout set. In order to evaluate the performance of an ML model, a study needs to be split into a train, validation, and holdout (test) set. Optimization is performed using the training and validation set, and the model that is ultimately used is being evaluated using the unseen holdout set. OmicLearn is intended to be an exploratory tool to assess the performance of algorithms when applied to specific datasets at hand rather than a classification model for production. Therefore, no holdout set is used, and the performance metrics have to be interpreted accordingly. This also prevents repeated analysis of the same dataset and choosing the same holdout set from leading to a selection bias and consequent over-interpretation of the model.
+
+## Other best practices
+We can recommend the following guideline for Machine Learning in Proteomics:
+ - [Interpretation of the DOME Recommendations for Machine Learning in Proteomics and Metabolomics](https://pubs.acs.org/doi/10.1021/acs.jproteome.1c00900)
+
+## Studies using OmicLearn
+To provide a perspective on the utility of OmicLearn, check out the following studies that used OmicLearn.
+
+- [High-resolution serum proteome trajectories in COVID-19 reveal patient-specific seroconversion](https://www.embopress.org/doi/full/10.15252/emmm.202114167)
+- [Proteome profiling of cerebrospinal fluid reveals biomarker candidates for Parkinson’s disease](https://doi.org/10.1016/j.xcrm.2022.100661)
```

Comment:
Could we also add the other two papers from external groups that cited OmicLearn? (especially the first one as it used transcriptome data from GEO and ArrayExpress rather than proteome. Even we could add two headlines Proteomics and Transcriptomics maybe?)

- [A 10-gene biosignature of tuberculosis treatment monitoring and treatment outcome prediction](https://doi.org/10.1016/j.tube.2021.102138)
- [Convergent cerebrospinal fluid proteomes and metabolic ontologies in humans and animal models of Rett syndrome](https://doi.org/10.1016/j.isci.2022.104966)

Reply:
Thanks Max! I now edited the page: [#6b32e9e396a0ce4109d0147dac28938d6275d747](https://github.com/MannLabs/OmicLearn/pull/2/commits/6b32e9e396a0ce4109d0147dac28938d6275d747)
For that publication, I could send the full-text!